### DISCUSSION TITLE
Reset Radio when i click submit button.

### DISCUSSION THREAD

**Discussion by ngodacthien** [shadcn-ui/ui#1612] [Q&A]
`if (step < 7) {
      const selectOptions = Array.from(
        document.querySelectorAll('input[name="point"]')
      ) as HTMLSelectElement[];
      selectOptions.forEach((selectOption) => {
        selectOption.selectedIndex = 0;
      });
      handleNextStep();
    }`
    
    `<RadioGroup
                      defaultValue="0"
                      className="flex"
                      name="point"
                      onChange={(e) => {
                        handleSelectChange(
                          index,
                          (e.target as HTMLInputElement).value
                        );
                      }}
                    >
                      {score.map((item) => (
                        <div
                          key={item.value}
                          className="flex items-center space-x-2"
                        >
                          <RadioGroupItem value={item.value} id={item.value} />
                          <Label htmlFor={item.label}>{item.value}</Label>
                        </div>
                      ))}
                    </RadioGroup> `
                    
how i can reset radio when click button submit

**Comment by Ezejaemmanuel** [comment]
i am also having the same problem here
